cli/cmd/root.go

```diff
@@ -31,30 +31,30 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initConfig)
 
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.cli.yaml)")
-	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "path to rundeck-zabbix config file")
 }
 
-// initConfig reads in config file and ENV variables if set.
+// initConfig reads in config file.
 func initConfig() {
-	if cfgFile != "" { // enable ability to specify config file via flag
+	if cfgFile != "" {
+		// enable ability to specify config file via flag
 		viper.SetConfigFile(cfgFile)
+	} else {
+		log.Error("no config file given.")
+		os.Exit(1)
 	}
 
-	// setup config file with defaults
-	if lib.FileExists(lib.ConfigPath) == false {
-		// only warn when not running setup command
-		log.Warn("Configuration file not found. Please setup using `setup` command")
-	} else {
-		viper.SetConfigName(lib.AppConfigName)   // name of config file (without extension)
-		viper.AddConfigPath(lib.ConfigDirectory) // adding home directory as first search path
-		viper.AutomaticEnv()                     // read in environment variables that match
+	if lib.FileExists(cfgFile) {
```

Comment:
if file doesn't exist && setupCmd.CalledAs() == "" throw the error else load configuration